features/crashprobe.feature

```diff
@@ -1,91 +1,81 @@
-
 Scenario: Executing privileged instruction
-    When I crash the app using "PrivilegedInstructionScenario"
-    And I relaunch the app
-    And I wait for a request
-    Then the request is valid for the error reporting API
-    And the "Bugsnag-API-Key" header equals "a35a2a72bd230ac0aa0f52715bbdc6aa"
-    And the payload field "notifier.name" equals "iOS Bugsnag Notifier"
-    And the payload field "events" is an array with 1 element
-    And the exception "errorClass" equals "EXC_BAD_ACCESS"
+    When I run "PrivilegedInstructionScenario" and relaunch the app
+    And I configure Bugsnag for "PrivilegedInstructionScenario"
+    And I wait to receive a request
+    Then the request is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    And the payload field "events" is an array with 1 elements
+    And the exception "errorClass" equals "EXC_BAD_INSTRUCTION"
     And the "method" of stack frame 0 equals "-[PrivilegedInstructionScenario run]"
 
 Scenario: Calling __builtin_trap()
-    When I crash the app using "BuiltinTrapScenario"
-    And I relaunch the app
-    And I wait for a request
-    Then the request is valid for the error reporting API
-    And the "Bugsnag-API-Key" header equals "a35a2a72bd230ac0aa0f52715bbdc6aa"
-    And the payload field "notifier.name" equals "iOS Bugsnag Notifier"
-    And the payload field "events" is an array with 1 element
-    And the exception "errorClass" equals "EXC_BAD_INSTRUCTION"
+    When I run "BuiltinTrapScenario" and relaunch the app
+    And I configure Bugsnag for "BuiltinTrapScenario"
+    And I wait to receive a request
+    Then the request is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    And the payload field "events" is an array with 1 elements
+    And the exception "errorClass" equals "EXC_BREAKPOINT"
     And the "method" of stack frame 0 equals "-[BuiltinTrapScenario run]"
 
 Scenario: Calling abort()
-    When I crash the app using "AbortScenario"
-    And I relaunch the app
-    And I wait for a request
-    Then the request is valid for the error reporting API
-    And the "Bugsnag-API-Key" header equals "a35a2a72bd230ac0aa0f52715bbdc6aa"
-    And the payload field "notifier.name" equals "iOS Bugsnag Notifier"
-    And the payload field "events" is an array with 1 element
+    When I run "AbortScenario" and relaunch the app
+    And I configure Bugsnag for "AbortScenario"
+    And I wait to receive a request
+    Then the request is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    And the payload field "events" is an array with 1 elements
     And the exception "errorClass" equals "SIGABRT"
     And the "method" of stack frame 0 equals "__pthread_kill"
-    And the "method" of stack frame 1 equals "abort"
-    And the "method" of stack frame 2 equals "-[AbortScenario run]"
+    And the "method" of stack frame 1 equals "<redacted>"
```

Comment:
Why "<redacted>"?  (Also in various other places in this file).

Reply:
Not quite sure I understand this one.  The change from "abort" to "<redacted>"?

Reply:
I think github is redacting our use of the word redacted!  But yes, why has it changed to redacted?